examples/drug_gripnet/README.md

```diff
@@ -1,13 +1,39 @@
-# Link Prediction: GripNet Framework on PoSE Sample Dataset
+# Polypharmacy Side Effect Prediction using GripNet
```

Comment:
Good to add after a dash or in ( ) to tell users that this is a *link prediction* problem.